packages/tabs/src/TabTitle.tsx

```diff
@@ -132,17 +150,23 @@ const TabTitle: ExtendableBox<BaseTabProps, 'button'> = ({
   ariaControl,
   index,
   darkMode,
+  containerNode,
   ...rest
 }: BaseTabProps) => {
   const { usingKeyboard: showFocus } = useUsingKeyboardContext();
   const titleRef = useRef<HTMLAnchorElement | HTMLButtonElement>(null);
   const mode = darkMode ? Mode.Dark : Mode.Light;
+  const activeEl = useDocumentActiveElement();
 
   useEffect(() => {
-    if (!disabled && selected && titleRef.current) {
-      titleRef.current.focus();
+    const tabsList = Array.from(containerNode?.current?.children || []);
+
+    if (activeEl !== null && tabsList.indexOf(activeEl) !== -1) {
+      if (!disabled && selected && titleRef.current) {
```

Comment:
it's gotten to the point where I think a comment to explain to the when and why of what's happening here would be helpful. I'm not really sure what we're trying to do here but can we achieve what we want by checking if `usePrevious(selected) !== selected`?

Reply:
What was going on is that we were prematurely (manually) setting a TabTitle as focused, so when a consuming application used Tabs and a user landed on a page, it would automatically scroll down to wherever the tabs were located (because the tabs were automatically focused). This PR adds a check to make sure that the TabTitles should receive focus before manually focusing them.